cypress/integration/SEPs.ts

```diff
@@ -546,12 +546,14 @@ context('Scientific evaluation panel tests', () => {
     cy.get('[role="dialog"]')
       .contains(sepMembers.reviewer.surname)
       .parent()
-      .find('[title="Add reviewer"]')
+      .find('input[type="checkbox"]')
```

Comment:
Why can't we keep using the title? I think using a title is easier to read and maintain

Reply:
I think the checkbox there has no "title", I can try to add a generic way where you can set it, it just the `PeopleTable` is used at many places already with this solution.